test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -872,6 +874,10 @@ void ConfigureTestServices(IServiceCollection services)
                         },
                     })
                 .Build();
+
+            services.AddOpenTelemetry()
```

Comment:
@ngruson Why is this needed?

Reply:
Hi, it doesn't seem to be necessary, I removed it and also looked at your other comments below.
But a seemingly unrelated test is failing now during the code coverage action.

```
Failed OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests.MockCollectorIntegrationTests.TestRecoveryAfterFailedExport [107 ms]
  Error Message:
   System.IO.IOException : Failed to bind to address http://localhost:4317.
```

Reply:
There is a known issue with `TestRecoveryAfterFailedExport `. Its all clear now.